api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -32,26 +32,76 @@ const (
 
 // IBMPowerVSClusterSpec defines the desired state of IBMPowerVSCluster.
 type IBMPowerVSClusterSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
 	// ServiceInstanceID is the id of the power cloud instance where the vsi instance will get deployed.
 	// +kubebuilder:validation:MinLength=1
+	// Deprecated: use ServiceInstance instead
 	ServiceInstanceID string `json:"serviceInstanceID"`
 
 	// Network is the reference to the Network to use for this cluster.
+	// Whenf the field is omitted, A DHCP service will be created in the service instance and its private network will be used.
 	Network IBMPowerVSResourceReference `json:"network"`
 
 	// ControlPlaneEndpoint represents the endpoint used to communicate with the control plane.
 	// +optional
```

Comment:
Are we going to allow user to provide this as well?
Or we are going to take it from the vpc load balancer by a GET call?
Please see whether this needs to added in the comment

Reply:
In case of using static ips as of today user can set  this.
Incase if user set this `controlPlaneLoadBalancer` then we will go ahead and provision loadbalancer and then update the values.

Reply:
In that case this can be in status instead of spec, what do you think?

Reply:
It cannot be in status because of two reasons
1. As per capi [contract](https://cluster-api.sigs.k8s.io/developer/providers/cluster-infrastructure#infracluster-resources) for infra providers it should be there in spec
2. Since its in spec user can set static ips and use the cluster as of today.